Cargo.toml

```diff
@@ -24,17 +24,20 @@ rust-version = "1.71.1"
 # security patching than updating statically linked software.
 
 [features]
-default = ["openssl", "vendored", "sha1"]
+default = ["dimpl", "vendored", "sha1"]
```

Comment:
vendored is tied to openssl so if dimpl is default then you don't need vendored?